src/app/Dashboard/DashboardSolo.tsx

```diff
@@ -51,7 +51,12 @@ const DashboardSolo: React.FC<DashboardSoloProps> = ({ ..._props }) => {
   const { search } = useLocation();
   const history = useHistory();
 
-  const cardConfigs: CardConfig[] = useSelector((state: RootState) => state.dashboardConfigs.list);
+  const dashboardConfigs = useSelector((state: RootState) => state.dashboardConfigs);
+
+  const cardConfigs: CardConfig[] = React.useMemo(
+    () => dashboardConfigs.layouts[dashboardConfigs.current].cards,
```

Comment:
Since now, cards are grouped by layouts. The index passing here should also be taken from query parameter. Maybe, `?layout=defaut&cardId=cardname-somehash`. Empty view should also mention this parameter.

This helps in the case where the user bookmark this url and comes back later.